muDIC/mesh/meshUtilities.py

```diff
@@ -259,6 +286,9 @@ def mesh(self, images, Xc1=0.0, Xc2=100.0, Yc1=0.0, Yc2=100., n_elx=4, n_ely=4,
 
             element = BSplineSurface(self.deg_e, self.deg_n, **kwargs)
 
+        elif self.type == "subset":
+            element = Q4()
```

Comment:
Not sure about this one right here... :) Please double check

Reply:
The code currently runs Q4 elements, even if the user specifies subset because of this tiny typo.